doc/source/whatsnew/v1.2.0.rst

```diff
@@ -315,7 +316,7 @@ Groupby/resample/rolling
 - Bug in :meth:`DataFrameGroupby.tshift` failing to raise ``ValueError`` when a frequency cannot be inferred for the index of a group (:issue:`35937`)
 - Bug in :meth:`DataFrame.groupby` does not always maintain column index name for ``any``, ``all``, ``bfill``, ``ffill``, ``shift`` (:issue:`29764`)
 - Bug in :meth:`DataFrameGroupBy.apply` raising error with ``np.nan`` group(s) when ``dropna=False`` (:issue:`35889`)
--
+- Fixed Dataframe Groupby value_counts with bins (:issue:`32471`)
```

Comment:
not sure what this means, so expand on this, and use a fully-qualified reference for value_counts
```:meth:`DataFrameGroupby.value_counts` ```